clients/src/main/java/org/apache/kafka/clients/consumer/Consumer.java

```diff
@@ -267,13 +269,92 @@ public interface Consumer<K, V> extends Closeable {
     void close();
 
     /**
+     * This method has been deprecated since Kafka 4.0 and should use {@link Consumer#close(CloseOption)} instead.
+     *
      * @see KafkaConsumer#close(Duration)
      */
+    @Deprecated
     void close(Duration timeout);
 
     /**
      * @see KafkaConsumer#wakeup()
      */
     void wakeup();
 
+    /**
+     * @see KafkaConsumer#close(CloseOption)
```

Comment:
I think we should write separate javadocs for this method because the default behavior when no GroupMembershipOperation is specified will be different. We can still reference the consumer javadocs and just add a note to point out the difference if you want

Reply:
oh sorry, I thought this was the Streams method...sorry about that, by mistake 😅